CHANGELOG.md

```diff
@@ -7,4 +7,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Example use case of spec-generated Ruby client. 
-[cybearl/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
+[cybeark/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
```

Comment:
s/cybeark/cyberark

Also, indent to match the indentation of the line above, starting at the word "Example"

Reply:
I believe I address these in the /authentication endpoint PR

Reply:
@telday in the `/authentication` endpoint PR you refer here, so neither of them really fix this problem. maybe submit a small PR just to fix this, and then you can rebase everything so they'll all be up to date?

Reply:
Good point. I didn't see the missing r as well. Just opened a PR